folly/container/detail/F14Table.h

```diff
@@ -247,16 +247,18 @@ FOLLY_ALWAYS_INLINE static void prefetchAddr(T const* ptr) {
   __builtin_prefetch(static_cast<void const*>(ptr));
 #elif FOLLY_NEON
   __prefetch(static_cast<void const*>(ptr));
-#else
+#elif FOLLY_SSE >= 2
   _mm_prefetch(
       static_cast<char const*>(static_cast<void const*>(ptr)), _MM_HINT_T0);
 #endif
 }
 
 #if FOLLY_NEON
 using TagVector = uint8x16_t;
-#else // SSE2
+#elif FOLLY_SSE >= 2
 using TagVector = __m128i;
+#else
+using TagVector = __uint128_t;
```

Comment:
This may need to be conditional on the uint128_t language extension being supported, though I think the only things that could reasonably hit this path should support the language extension just fine.

Reply:
I've added the `FOLLY_HAVE_INT128_T` check